src/Assetic/Filter/BaseProcessFilter.php

```diff
@@ -35,11 +35,11 @@ public function setTimeout($timeout)
      *
      * @param array $arguments An optional array of arguments
      *
-     * @return ProcessBuilder A new process builder
+     * @return Process A new process builder
```

Comment:
that's a BC break, so cannot be done in minor versions of Assetic.

Reply:
Agreed, but it's not my decision how you tag this afterwards.